### ISSUE TITLE
v5.8.8 - "symfony console make:migration" is broken 

### ISSUE THREAD

**Issue by mihaikelemen** [symfony-cli/symfony-cli#438]
Framework version:  v6.4.3 with MariaDb running in Docker. I have the proper connection settings made (server version & charset)

Running `symfony console make:migration` fails to work properly and the error reported is 

~~~text
The metadata storage is not up to date, please run the sync-metadata-storage command to fix this issue. 
~~~

Running `./bin/console make:migration`  works as expected. Downgrading to v5.8.7 works also.

**Comment by fabpot** [comment]
Can you share the output of `symfony var:export DATABASE_URL`? And maybe the content of `config/packages/doctrine.yaml` and `compose.yaml` as well.

**Comment by mihaikelemen** [comment]
Sure thing.

Output for `symfony var:export DATABASE_URL`
~~~text
mysql://DBUSER:DBPASS@127.0.0.1:3306/DBNAME?sslmode=disable&charset=utf8mb4&serverVersion=10.11.2+maria~ubu2204
~~~

Configuration under `./config/packages/doctrine.yaml`

~~~yml
doctrine:
    dbal:
        url: '%env(resolve:DATABASE_URL)%'
        server_version: '10.11.2-MariaDB-1:10.11.2+maria~ubu2204'
        charset: utf8mb4
        default_table_options:
            charset: utf8mb4
            collate: utf8mb4_unicode_ci
        mapping_types:
            enum: string

        # IMPORTANT: You MUST configure your server version,
        # either here or in the DATABASE_URL env var (see .env file)
        #server_version: '15'
    orm:
        auto_generate_proxy_classes: true
        enable_lazy_ghost_objects: true
        naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
        auto_mapping: true
        mappings:
            App:
                is_bundle: false
                dir: '%kernel.project_dir%/src/Entity'
                prefix: 'App\Entity'
                alias: App

when@test:
    doctrine:
        dbal:
            # "TEST_TOKEN" is typically set by ParaTest
            dbname_suffix: '_test%env(default::TEST_TOKEN)%'

when@prod:
    doctrine:
        orm:
            auto_generate_proxy_classes: false
            proxy_dir: '%kernel.build_dir%/doctrine/orm/Proxies'
            query_cache_driver:
                type: pool
                pool: doctrine.system_cache_pool
            result_cache_driver:
                type: pool
                pool: doctrine.result_cache_pool

    framework:
        cache:
            pools:
                doctrine.result_cache_pool:
                    adapter: cache.app
                doctrine.system_cache_pool:
                    adapter: cache.system
~~~

Contents of `docker-compose.yml`

~~~yml
version: '3'

networks:
    app_network:

services:
    database:
        image: mariadb:10.11
        container_name: '${APP_NAME}-db'
        restart: always
        ports:
            - '${DB_PORT}:3306'
        environment:
            MYSQL_ROOT_PASSWORD: '${MYSQL_ROOT_PASSWORD}'
            MYSQL_USER: '${MYSQL_USER}'
            MYSQL_PASSWORD: '${MYSQL_PASSWORD}'
            MYSQL_DATABASE: '${MYSQL_DATABASE}'
        volumes:
            - db_data:/var/lib/mysql
            - ./docker/database/mariadb/my.cnf:/etc/mysql/conf.d/my.cnf
        healthcheck:
            test: mysqladmin ping -h 127.0.0.1 -u root --password=$$MYSQL_ROOT_PASSWORD
            interval: 5s
            retries: 5
        networks:
            - app_network

    mailer:
        container_name: '${APP_NAME}-mailer'

volumes:
    db_data:
~~~

**Comment by fabpot** [comment]
Everything looks fine to me. It's probably related to this: https://github.com/doctrine/DoctrineMigrationsBundle/issues/337#issuecomment-645390496

**Comment by mihaikelemen** [comment]
For the moment I will stick to the previous release (eg. v5.8.7).
Thanks.

**Comment by fabpot** [comment]
Should be fixed by #439

**Comment by mihaikelemen** [comment]
Thank you.
